### PR TITLE
Display related tags on articles

### PR DESCRIPTION
**Pull request by ironsidevsquincy**
Use the same tag decision logic as used in streams

screen shot 2015-04-14 at 12 08 51

(Only display on L+)

### CODE REVIEW

File: server/controllers/capi.js [hunk 1]
```diff
@@ -134,12 +134,31 @@ module.exports = function(req, res, next) {
...
-					var isColumnist;
 					// Some posts (e.g. FastFT are only available in CAPI v2)
-					if (articleV1) {
-						// TODO: Replace with something in CAPI v2
-						isColumnist = articleV1.item.metadata.primarySection.term.name === 'Columnists';
-					}
+					// TODO: Replace with something in CAPI v2
+					var isColumnist = articleV1 && articleV1.item.metadata.primarySection.term.name === 'Columnists';
+
+					var articleV1Metadata = articleV1 && articleV1.item.metadata;
```
Comment (matthew-andrews) [inline server/controllers/capi.js]: sorry to be a pain but can we have a `mentionsV2` implementation too?  (Use V2 metadata and point semantic streams — organisations/uuid & people/uuid)

Comment (ironsidevsquincy) [inline server/controllers/capi.js]: yep, of course

**Comment by ironsidevsquincy** [comment]
@matthew-andrews, not showing first 5 annotations if `mentionsV2 ` is on (though this is pretty much what we get at the bottom of the article)

**Comment by matthew-andrews** [comment]
cool

**Pull request merged**